services/app-web/src/s3/s3Amplify.ts

```diff
@@ -47,18 +47,18 @@ function newAmplifyS3Client(bucketConfig: S3BucketConfigType): S3ClientT {
             file: File,
             bucket: BucketShortName
         ): Promise<string | S3Error> => {
+            const fileHash = await calculateSHA256(file)
```

Comment:
does this mean that we are running calculateSHA256 twice? once to put in Proto and once to put in S3?

Reply:
After discussion, we wound up not writing the sha to S3 tags, so this code has been removed.  Because we're going to add it to the document, and that document will soon be moved out of the proto and become queryable, it seemed redundant and potentially confusing to also store the sha in S3.